draft-ietf-quic-transport.md

```diff
@@ -1461,127 +1461,126 @@ connection if the integrity check fails with a PROTOCOL_VIOLATION error code.
+packets.  The PATH_CHALLENGE frame MAY be bundled with other frames, but this
+packet MUST be padded to exactly 1200 octets unless the client has a reasonable
+assurance that the PMTU is larger.  Sending a packet of this size confirms that
+the network path from the client to the server supports an MTU of this size and
+helps reduce the amplitude of amplification attacks caused by server responses
+toward an unverified client address.
+
+A client MAY send additional PATH_CHALLENGE frames to handle packet loss or to
+get more measurements on the new network path, subject to the congestion
+controller's limits. The client SHOULD use fresh random data in each
+PATH_CHALLENGE frame so that it can associate the server's response with the
+causative PATH_CHALLENGE.
+
+When the client receives a PATH_RESPONSE from the server and verifies that the
+included data matches that sent by the client in a previous PATH_CHALLENGE, the
+server is considered reachable from the new address and the PMTU of the new
+network path is considered verified.
+
+A client SHOULD abandon the new address when it does not receive a PATH_RESPONSE
+after sending some number of PATH_CHALLENGE frames and/or after some time has
+passed. Note that the client may be receiving data from the server during this
+period, but not receiving a PATH_RESPONSE may be indicative of a PMTU detection
+failure.
+
+A client may receive a PATH challenge from a server that is validating the
+client's ownership of an address. A client responds to this frame by echoing the
+data from the PATH_CHALLENGE frame in a PATH_RESPONSE frame.  This response MAY
+be bundled with other frames and does not require additional padding.
+
+
+### Server Behavior {{migration-server}}
+
+When a server receives a PATH_CHALLENGE frame from a client, it MUST respond by
+copying the included data in a PATH_RESPONSE frame, and send it in a packet
+padded to the size of the packet carrying the PATH_CHALLENGE. This PATH_RESPONSE
+MUST be sent to the address that the PATH_CHALLENGE was received from.
+
+The server need not retransmit its PATH_RESPONSE, since an interested client is
+expected to send more PATH_CHALLENGE frames when a response is not received for
+a prior one.
+
+Before sending any other packets to a client address that is not yet considered
+valid, the server MUST send a PATH_CHALLENGE frame with its own validation data
+to verify the client's ownership of this address. The server's PATH_CHALLENGE
+MAY be bundled with other frames in a packet that does not need to be padded.
+
+When the server receives a PATH_RESPONSE from the client and verifies that the
+included data matches that sent by the server in a previous PATH_CHALLENGE, the
+server considers the client's address to which the corresponding PATH_CHALLENGE
+was sent as valid.
+
+A server MAY send additional PATH_CHALLENGE frames to handle packet loss,
+subject to the congestion controller's limits. The server MUST use fresh random
```

Comment:
Why is this a MUST for server, and a SHOULD for clients?

Reply:
Because there are potential security implications for the server but not for the client.

Reply:
Should we have a line explaining what the security concern is?

Reply:
I've added some text